src/components/VariantDb/StatsTable.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from 'react';
+import { Col, List, Row } from 'antd';
+import Title from 'antd/es/typography/Title';
+import { fetchVariantDBStats } from './fetchVariantStats';
+
+type VariantDbStats = {
+  distinctVariantsCount: number;
+  occurrencesCount: number;
+  participantsCount: number;
+  studiesCount: number;
+};
+
+const initialVariantStatsState = [
+  {
+    name: 'Studies',
+    value: 0,
+  },
+  {
+    name: 'Participants',
+    value: <div>-</div>,
+  },
+  {
+    name: 'Distinct Variants',
+    value: 0,
+  },
+  {
+    name: 'Occurrences',
+    value: 0,
+  },
+];
+
+const StatsTable = (): JSX.Element => {
+  const [statsData, setStatsData] = useState(initialVariantStatsState);
+
+  useEffect(() => {
+    async function fetchStats() {
+      try {
+        const stats = await fetchVariantDBStats();
```

Comment:
I don't think that the api should return _source but rather only the needed data.